tests/integration/pilot/validation_test.go

```diff
@@ -78,10 +78,8 @@ func TestValidation(t *testing.T) {
 				// We are only checking the string literals of the rejection reasons
 				// from the webhook and the k8s api server as the returned errors are not
 				// k8s typed errors.
-				return strings.Contains(err.Error(), "denied the request") ||
-					strings.Contains(err.Error(), "error validating data") ||
-					strings.Contains(err.Error(), "Invalid value") ||
-					strings.Contains(err.Error(), "is invalid")
+				// Note: this explicitly does NOT catch OpenAPI schema rejections - only validating webhook rejections.
+				return strings.Contains(err.Error(), "denied the request")
```

Comment:
surprised removing these didn't cause other validation cases to start failing (or else why were they there in the first place?)

Reply:
I am 90% the other ones were (incorrectly) added to catch open api schema rejections - when we explicitly only want webhook rejections. The 10% chance is that other versions of kubernetes had different error message...